tests/mpls_test.py

```diff
@@ -23,71 +25,203 @@
 Vlan2         enable
 """
 
+show_interfaces_mpls_output_1="""\
+Interface     MPLS State
+------------  ------------
+Ethernet2     enable
+Ethernet4     enable
+Ethernet8     disable
+Ethernet16    disable
+Loopback0     disable
+PortChannel2  disable
+Vlan2         enable
+"""
+
 show_interfaces_mpls_specific_output="""\
 Interface    MPLS State
 -----------  ------------
-Ethernet2    enable
+Ethernet4    disable
+"""
+
+show_interfaces_mpls_output_frontend="""\
+Interface    MPLS State
+-----------  ------------
+Ethernet16   enable
+Ethernet20   enable
+Ethernet28   disable
+"""
+
+show_interfaces_mpls_output_all="""\
+Interface    MPLS State
+-----------  ------------
+Ethernet16   enable
+Ethernet20   enable
+Ethernet28   disable
+Ethernet128  disable
 """
 
+show_interfaces_mpls_masic_output_frontend="""\
+Interface    MPLS State
+-----------  ------------
+Ethernet0    enable
+Ethernet4    disable
+"""
+
+show_interfaces_mpls_masic_output_all="""\
+Interface       MPLS State
+--------------  ------------
+Ethernet0       enable
+Ethernet4       disable
+Ethernet64      enable
+Ethernet-BP0    enable
+Ethernet-BP4    disable
+Ethernet-BP256  disable
+Ethernet-BP260  enable
+"""
+show_interfaces_mpls_masic_output_asic_all="""\
+Interface     MPLS State
+------------  ------------
+Ethernet0     enable
+Ethernet4     disable
+Ethernet-BP0  enable
+Ethernet-BP4  disable
+"""
+
+
+
+
 modules_path = os.path.join(os.path.dirname(__file__), "..")
 test_path = os.path.join(modules_path, "tests")
+scripts_path = os.path.join(modules_path, "scripts")
 sys.path.insert(0, modules_path)
 sys.path.insert(0, test_path)
 mock_db_path = os.path.join(test_path, "mpls_input")
+masic_mock_db_path = os.path.join(test_path, "mock_tables")
+masic0_mock_db_path = os.path.join(masic_mock_db_path, "asic0")
 
 
 class TestMpls(object):
     @classmethod
     def setup_class(cls):
         print("SETUP")
-        os.environ['UTILITIES_UNIT_TESTING'] = "1"
```

Comment:
why remove this line. Some scripts depend on the env variable to load mock APIs

Reply:
sure